packages/coin-support-utils/tests/01.common/01.mapDerivationPath/__fixtures__/valid.ts

```diff
@@ -0,0 +1,19 @@
+import { MapDerivationPathTestCases } from './types';
+
+export const valid: MapDerivationPathTestCases[] = [
+  {
+    name: 'should map derivation path with single index',
+    input: "m/44'",
+    output: [2147483692],
```

Comment:
Write the output in terms of `0x80000000` to make it more readable.

```suggestion
    output: [0x80000000 + 44],
```